persistence/sql/sqlsaga.md

```diff
@@ -0,0 +1,47 @@
+---
+title: SqlSaga Base Class
+component: SqlPersistence
+related:
+ - persistence/sql/saga
+reviewed: 2018-06-08
+---
+
+`SqlSaga<T>` is an alternate, experimental saga base class for use with SQL Persistence that offers a less verbose mapping API than `NServiceBus.Saga<T>`. It is possible (but not guaranteed) that in the future, the NServiceBus saga class will evolve to look more like `SqlSaga`. Which API to use is largely a matter of personal preference.
```

Comment:
> `SqlSaga<T>` is an alternate, experimental saga base class

If I understood correctly, for now we're going to fully support both as equal citizens, but this phrase indicates a different thing and may raise unnecessary concerns among people who currently are using SqlSagas.

> It is possible (but not guaranteed) that in the future, 

Skip this part since we cannot make any promises here.